SPECS/expat/CVE-2022-40674.patch

```diff
@@ -0,0 +1,49 @@
+From 4a32da87e931ba54393d465bb77c40b5c33d343b Mon Sep 17 00:00:00 2001
```

Comment:
Is the CVE patch still needed if we upgrade expat to 2.4.9?
From the release notes, it seems like the patch is already incorporated into 2.4.9.

Reply:
deleted patch file